fsproxy/src/test/java/no/unit/identityservice/fsproxy/FsApiTest.java

```diff
@@ -0,0 +1,79 @@
+package no.unit.identityservice.fsproxy;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.core.Is.is;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import java.io.IOException;
+import java.nio.file.attribute.UserPrincipalNotFoundException;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class FsApiTest {
+
+    private FsApi fsApi;
+    private FsMock fsMock;
+
+    @AfterEach
+    public void tearDown() {
+        fsMock.shutDown();
+    }
+
+    @BeforeEach
+    void init() {
+        fsMock = new FsMock();
+        fsMock.initialize();
```

Comment:
The initialize() method should be called inside the constructor of FsMock().

Reply:
fixed